safe-global/safe-dao-governance-app
# [EPIC] SEP #5 - Hide after redeem

## What it solves

Hides SEP #5-related elements after redemption.

## How this PR fixes it

The widget outline/badges and info boxes related to SEP #5 on the intro/claim pages are now hidden if the qualifier has already redeemed.

## How to test it

1. Open the test Safe (`0x5f310dc66F4ecDE9a1769f1B7D75224dA592201e`).
2. Add this PR deployment as a custom app.
3. Observe the allocation of `69.42` from new test aidrop (`0xf7ec19587A59af367924c301C9b25A02A0BA6b73`).
4. Observe that the aforementioned elements are no longer visible after redemption.

## Screenshots

Compared to the screenshots from https://github.com/safe-global/safe-dao-governance-app/pull/35:

![image](https://github.com/safe-global/safe-dao-governance-app/assets/20442784/9ddd496d-6e2a-45a4-94be-150f83dfd244)

![image](https://github.com/safe-global/safe-dao-governance-app/assets/20442784/e3f63ffb-2c2a-445a-ac61-0d519bd50919)

![image](https://github.com/safe-global/safe-dao-governance-app/assets/20442784/35b450a1-d489-4496-98f8-737bfd90b153)

## Checklist
